fsspec/compression.py

```diff
@@ -139,7 +139,7 @@ def _fetch_range(self, start, end):
 try:
     import snappy
 
-    snappy.compress
+    snappy.compress  # noqa: B018
```

Comment:
We could call this on something
```suggestion
    snappy.compress(b"0")
```

Reply:
Perhaps even `b""`?

Reply:
Apparently b"" is fine - I had assumed it needed some content.